tracer/test/snapshots/AspNetWebApi2TestsModuleOnly.__path=_api_transient-failure_false_statusCode=500.verified.txt

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: CustomTracingExceptionHandler.handle-async,
+    Resource: CustomTracingExceptionHandler.handle-async,
+    Service: sample,
+    ParentId: Id_3,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      span.kind: server,
+      version: 1.0.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
+    Name: aspnet.request,
+    Resource: GET /api/transient-failure/false,
```

Comment:
FYI another interesting case where the resource name differs because we don't run middleware: Instead of `GET /api/transient-failure/{value}` we get `GET /api/transient-failure/false`